R/C-flux/description.R

```diff
@@ -6,22 +6,123 @@ get_file(node = "pk4bg",
          path = "data/C-Flux/summer_2020",
          remote_path = "C-Flux")
 
-flux.colnames <- read_csv("Three-D_c-flux_2020.csv") %>% 
+flux <- read_csv("Three-D_c-flux_2020.csv")
+
+flux.colnames <- flux %>% 
   colnames()
 
-variables <- tibble(flux.colnames, description = c(
+variables <- tibble(flux.colnames, "Variable type" = c(
+  # Datetime
```

Comment:
can you do this automatically, by using class()?

Reply:
See my message on slack